src/main/java/org/folio/circulation/domain/Item.java

```diff
@@ -145,17 +151,31 @@ public String getInstanceId() {
   }
 
   public String getCallNumber() {
-    return getProperty(holdingRepresentation, "callNumber");
+    return getEffectiveCallNumbers(CALL_NUMBER_KEY);
   }
 
   public String getCallNumberPrefix() {
-    return getProperty(holdingRepresentation, "callNumberPrefix");
+    return getEffectiveCallNumbers(CALL_NUMBER_PREFIX_KEY);
   }
 
   public String getCallNumberSuffix() {
-    return getProperty(holdingRepresentation, "callNumberSuffix");
+    return getEffectiveCallNumbers(CALL_NUMBER_SUFFIX_KEY);
+  }
+
+  private String getEffectiveCallNumbers(String propertyName) {
+    return hasItemRepresentationCallNumber()
+      ? getItemRepresentationCallNumbers(propertyName)
+      : getProperty(holdingRepresentation, propertyName);
+  }
+
+  private boolean hasItemRepresentationCallNumber() {
+    return StringUtils.isNotBlank(getItemRepresentationCallNumbers(CALL_NUMBER_KEY));
   }
 
+  private String getItemRepresentationCallNumbers(String propertyName) {
+    return getProperty(itemRepresentation,
+      String.format(ITEM_REPRESENTATION_PREFIX, StringUtils.capitalize(propertyName)));
```

Comment:
Rather than attempting to generate the item property name from the holding property name, could these constants be defined separately for the two kinds of records?

Reply:
If it makes sense to create a `HoldingsProperties` class to aid this purpose, then I think that is ok

Reply:
done